src/components/Topbar.jsx

```diff
@@ -2,16 +2,29 @@ import styles from '../styles/topbar'
 
 import React from 'react'
 import { translate } from '../lib/I18n'
+import { withRouter } from 'react-router'
 
 import Toolbar from '../containers/Toolbar'
 
-export const Topbar = ({ t, viewName }) => {
+export const Topbar = ({ t, router, viewName, albumName = '' }) => {
+  const backToAlbums = () => {
+    // go to parent
+    let url = router.location.pathname
+    router.push(url.substring(0, url.lastIndexOf('/')))
+  }
   return <div className={styles['pho-content-header']}>
+    {viewName === 'albumContent' &&
+      <div
```

Comment:
Correct, but it would be implicit and not require the `withRouter` HOC. I think this is desirable, but it might be a personal preference.